Assets/Scripts/Models/Parameter.cs

```diff
@@ -233,7 +230,7 @@ public void WriteXmlParamGroup(XmlWriter writer)
             writer.WriteStartElement("Param");
             writer.WriteAttributeString("name", name);
         }
-        if (!(value == null) )
+        if ((value != null) )
```

Comment:
I'm dumb... I was just reading it as you correcting me on what it was before... *sigh*